lib/test/CMakeLists.txt

```diff
@@ -656,6 +656,10 @@ endif ()
 ot_check_test (ANCOVA_std)
 ot_check_test (FAST_std)
 ot_check_test (SobolIndicesAlgorithm_std)
+ot_check_test (HSICStat_std IGNOREOUT)
+ot_check_test (HSICEstimatorConditionalSensitivity_std IGNOREOUT)
```

Comment:
careful you still got a lib/test/t_HSICEstimatorGlobalSensitivity_std.expout file

Reply:
Thanks! I've removed it now.